features/users/authenticate_users.feature

```diff
@@ -226,22 +226,21 @@ Feature: User Authentication
     Then I should see "Successfully logged in."
       And I should see "You'll stay logged in for 2 weeks even if you close your browser"
 
-  # TODO make this an actual test - it's been 4 years...
-  Scenario Outline: Show or hide preferences link
-    Given I have no users
-      And the following activated users exist
-      | login    | password |
-      | sam      | secret   |
-      | dean     | secret   |
-    And I am logged in as "<login>" with password "secret"
-    When I am on <user>'s user page
-    Then I should <action>
-
-    Examples:
-      | login | user  | action                   |
-      | sam   | sam   | not see "Log In"         |
-      | sam   | sam   | see "Log Out"            |
-      | sam   | sam   | see "Preferences" within "#dashboard"    |
-      | sam   | dean  | see "Log Out"            |
-      | sam   | dean  | not see "Preferences" within "#dashboard" |
-      | sam   | dean  | not see "Log In"         |
+  Scenario: Passwords cannot be reset for users who have been given the protected role due to trolling or harassment.
```

Comment:
There's already a test for actually assigning roles in admin_manage_users.feature, but I can go through the whole process here as well if we'd rather.

https://github.com/otwcode/otwarchive/blob/398a1f8c20238fa1a7ca0c12d3b1367c974d84da/features/admins/users/admin_manage_users.feature#L28-L43

Reply:
No, I think it's fine to leave testing role assignment to that other file.